### PR TITLE
Support for PHPUnit 7

### PR DESCRIPTION
**Pull request by oshdev**
Hey, I know that some of the changes from this PR are not acceptable for this repo. There's also no tests fix.
However the solution does _(seem to)_ work, so feel free to cherry pick it from this if you like. Unfortunately I don't have time right now and I don't expect any time soon to be able to amend the changes to be universal and even more so to fix the tests.
Best regards.

### CODE REVIEW

File: src/Printer.php [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
+use function explode;
+use function floor;
+use function is_callable;
+use function is_string;
+use function round;
+use function sprintf;
+use function str_repeat;
+use function strlen;
+use function strpos;
+use function substr;
```
Comment (Bilge) [inline src/Printer.php:21]: This is absolutely shocking.

**Comment by Bilge** [comment]
Dumping a PR in this state isn't particularly helpful. If you don't intend to work on it then I should just close it.

If you *do* intend to work on it then the PR needs some commentary on the challenges faced supporting PHPUnit 7 and how this PR addresses them.

File: src/Printer.php (hunk 1 above)
Comment (dkraczkowski) [reply src/Printer.php:21]: Well this is standard nowadays.

Comment (oshdev) [reply src/Printer.php:21]: You are always easily shocked Paul :)
Not trying to convince you, but here's a read if you're not aware: https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/

Comment (Bilge) [reply src/Printer.php:21]: As the article clearly states in the final example, the solution can be expressed in one of two forms. Unfortunately, you elected to use the wrong form.

Comment (oshdev) [reply src/Printer.php:21]: As usual you express your opinions as facts. I just adhere to Doctrine coding standard here.

**Pull request closed without merging**